scripts/.htaccess

```diff
@@ -2,3 +2,10 @@ php_flag display_errors on
 
 # Disable access to this directory via web.
 Deny from all
+
+# In case we change our mind about the above, deny access to sql scripts in this directory
+# regardless.
```

Comment:
🎉

Reply:
Also, the result of me doing the branch before I realized how bad the actual situation was and removed them from master :)